### PR TITLE
Better keycode management & small cleanup

### PR DESCRIPTION
**Pull request by manniL**
**What kind of change does this PR introduce?**

Refactoring code to clean it up a little

* Use destructuring (just one case) over X-line assignments
* Add a keycodes file to store consts instead of having comments all over the place

**Checklist**:
- [ ] Documentation
- [ ] Tests
- [ ] Ready to be merged
- [x] Added myself to contributors table

### CODE REVIEW

**Comment by SaraVieira** [comment]
I'mma merge it!

Thank you so much for helping out!

File: packages/homepage/src/pages/explore.js [hunk 1]
```diff
@@ -74,8 +74,7 @@ export default class Explore extends React.PureComponent {
   openSandbox = index => {
     const sandbox = this.state.sandboxes[index];
     const { id, picks } = sandbox;
-    const title = picks[0].title;
-    const description = picks[0].description;
+    const { title, description } = picks[0];
```
Comment (donavon) [inline packages/homepage/src/pages/explore.js]: or... 😉

```js
 const [{ title, description }] = picks;
```

Comment (manniL) [reply packages/homepage/src/pages/explore.js]: Even cleaner :record_button: :+1:

Comment (donavon) [reply packages/homepage/src/pages/explore.js]: i just hate seeing array access with numeric literals when you can avoid it. it's one of my red flags when i'm reviewing at code.

**Pull request merged**